src/InboundServer/api.yaml

```diff
@@ -1,4427 +1,4533 @@
-openapi: 3.0.0
+swagger: "2.0"
```

Comment:
@vgenev - thanks for the PR; I think to help the reviews, we need to keep this to openapi 3 and just do the changes we need for v1.1 so that the changes are manageable

here's the openapi3 version of v1.1 FSPIOP API

Reply:
+1